packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -44,6 +46,10 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
     const indexOfSelection = selected_country =>
         residence_list.findIndex(item => item.text.toLowerCase() === selected_country.toLowerCase());
 
+    const checkBoxRequired = selected_country => [localize('Brazil')].includes(selected_country);
+
```

Comment:
Since hint text is specifically says Brazil, could we use more region-specific variable names in this context?

Reply:
ok, will tailor it specifically to brazil. may be in the future if required, we can rename it..